devpi-template.yaml

```diff
@@ -0,0 +1,294 @@
+apiVersion: v1
+kind: Template
+labels:
+  template: devpi
+metadata:
+  name: devpi
+  annotations:
+    description: deploys devpi with nginx fronend and other relevant stuff
+    tags: devpi, private-pypi, dev-pypi
+objects:
+- apiVersion: v1
+  kind: PersistentVolumeClaim
+  metadata:
+    name: "devpi-db-storage-pvc"
+  spec:
+    accessModes:
+    - ReadWriteOnce
+    resources:
+      requests:
+        storage: "1Gi"
+    volumeName: "${NAME}-db-storage"
+
+- apiVersion: v1
+  kind: PersistentVolumeClaim
+  metadata:
+    name: "${NAME}-file-storage-pvc"
+  spec:
+    accessModes:
+    - ReadWriteMany
+    resources:
+      requests:
+        storage: "10Gi"
+    volumeName: "devpi-file-storage"
+
+- apiVersion: v1
+  kind: Route
+  metadata:
+    name: "${NAME}-frontend"
+  spec:
+    host: "${FRONTEND_URL}"
+    port:
+      targetPort: 80
+    to:
+      kind: Service
+      name: "nginx"
+
+- apiVersion: v1
+  kind: Service
+  metadata:
+    name: "nginx"
+    annotations:
+      description: waits for pypi requests and serves them
+  spec:
+    ports:
+    - name: "http"
+      port: 80
+      targetPort: 80
+    selector:
+      name: "nginx"
+
+- apiVersion: v1
+  kind: ConfigMap
+  metadata:
+    name: nginx-configs
+  data:
+    nginx-devpi.conf: |
```

Comment:
id like to put this key back into a own file as soon as we have a sensible way to sync or pull it in